### PR TITLE
Header improvements

### PR DESCRIPTION
**Pull request by lopenchi**

### CODE REVIEW

File: css/components/header/in-site/icon.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.header__in-site__icon {
+    height: 52px;
+    display: inline-flex;
+    align-items: center;
```
Comment (well1791) [inline css/components/header/in-site/icon.css:4]: awesome!

File: css/components/header/in-site/icon.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.header__in-site__icon {
+    height: 52px;
```
Comment (well1791) [inline css/components/header/in-site/icon.css:2]: use `block-size: 3.25rem` that should do it, only if your browser uses the browser default font size (16px)

File: css/components/header/in-site/list.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    }
+}
+
+.header__in-site__icon:focus-within + .header__in-site__list {
+    display: block;
+    position: absolute;
+    max-width: 120px;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+    margin-top: 40px;
+    z-index: 1
```
Comment (well1791) [inline css/components/header/in-site/list.css]: my advice here, try not to use z-index only if nothing else works

File: css/components/header/in-site/list.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    .header__in-site__list {
+        display: flex;
+    }
+}
+
+.header__in-site__icon:focus-within + .header__in-site__list {
+    display: block;
+    position: absolute;
+    max-width: 120px;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
```
Comment (well1791) [inline css/components/header/in-site/list.css]: please use a variable for that `rgba`

File: css/elements/commons.css [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+button {
+    height: 52px;
+    cursor: pointer;
+}
+
+input {
+    height: 52px;
+}
```
Comment (well1791) [inline css/elements/commons.css:17]: good naming, although I would change the use of `px` for `rem` and define a variable for that, maybe `--min-action-height` and then `block-size: var(--min-action-height)`

File: css/components/header/in-site/list.css (hunk 3 above)
Comment (lopenchi) [reply css/components/header/in-site/list.css]: I remove it, and it continue working, Thanks for the advice

File: css/components/header/in-site/list.css (hunk 4 above)
Comment (lopenchi) [reply css/components/header/in-site/list.css]: Changing in other PR with all variables

File: css/elements/commons.css (hunk 5 above)
Comment (lopenchi) [reply css/elements/commons.css:17]: Doing in next PR

**Pull request merged**